src/serde_value_manager.rs

```diff
@@ -222,14 +222,30 @@ impl<'a> WriteHolder<Value, Value> for KeyHolderWrite<'a> {
             self.set_root(Some(v))?;
             updated = true;
         } else {
-            update(&path, self.get_value().unwrap().unwrap(), |_v| {
+            replace(&path, self.get_value()?.unwrap(), |_v| {
                 updated = true;
                 Ok(Some(v.take()))
             })?;
         }
         Ok(updated)
     }
 
+    fn merge_value(&mut self, path: Vec<String>, v: Value) -> Result<bool, RedisError> {
```

Comment:
Not using `json-patch` crate (Ref #150) since it only works with serde_json `Value` and not wiith `IValue`?

Reply:
right, this is why I had to implement merge myself